Automattic/jetpack
# Autoloader: Revert the Upgrade Handler

Revert https://github.com/Automattic/jetpack/pull/23039

WooCommerce also uses the autoloader, but is including files from an upstream that extends non-present classes. As an example: https://plugins.trac.wordpress.org/browser/woocommerce/tags/6.3.0/vendor/composer/jetpack_autoload_classmap.php#L53

This is also what led to a couple other fatals for similar reasons. I think it isn't solid enough if the risk of fataling is this high so should figure out something else, sadly.

#### Changes proposed in this Pull Request:

* Comment out new code.

#### Jetpack product discussion


p1646778795184589-slack-C01U2KGS2PQ

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* With Jetpack 10.8-a.3, download an older version of WooCommerce (6.2.x) and connect it.
* Upgrade to WooCommerce 6.3. See fatal.
* With this branch, repeat.
